o19s/opensearch-search-quality-evaluation
# Easy to install Quality Evaluation Framework dashboards

This PR creates a shell script that installs the Quality Evaluation Framework dashboards on a fresh installation of Open Search Dashboards.

The shell script has no dependencies other than curl and a running installation of Open Search and Open Search Dashboards.

The script also uploads sample data to allow the user to get to know the dashboards.

In order to make the dashboards visible you may need to extend the time window that is being considered. You can look at the last 15 weeks for example.

To test the installation run:
```
./install_dashboards.sh
```

You will see output such as the following:
```
Using Open Search and Open Search Dashboards at localhost:9200 and localhost:5601 respectively.
  (set environment variables OPEN_SEARCH and OPEN_SEARCH_DASHBOARDS otherwise)

Deleting index sqe_metrics_sample_data
Populating index sqe_metrics_sample_data with sample metric data

Installing Quality Evaluation Framework Dashboards
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 45873  100  2425  100 43448   2474  44339 --:--:-- --:--:-- --:--:-- 46856
The Quality Evaluation Framework Dashboards were successfully installed

```
